### ISSUE TITLE
[Build error] cl65: Don't know what to do with `100'

### ISSUE THREAD

**Issue by lifepillar** [datajerk/c2t#3]
When I try to build both v0.996 and the current master, I get the following error:

```
cl65 -t none --listing --list-bytes 100 autoload.s 
cl65: Don't know what to do with `100'
make[1]: *** [autoload] Error 1
hexdump: autoload: No such file or directory
hexdump: autoload: Bad file descriptor
hexdump: autoload: No such file or directory
hexdump: autoload: Bad file descriptor
hexdump: inflate: No such file or directory
hexdump: inflate: Bad file descriptor
hexdump: inflate: No such file or directory
hexdump: inflate: Bad file descriptor
hexdump: fastload9600: No such file or directory
hexdump: fastload9600: Bad file descriptor
hexdump: fastload9600: No such file or directory
hexdump: fastload9600: Bad file descriptor
hexdump: fastload8000: No such file or directory
hexdump: fastload8000: Bad file descriptor
hexdump: fastload8000: No such file or directory
hexdump: fastload8000: Bad file descriptor
hexdump: fastloadcd: No such file or directory
hexdump: fastloadcd: Bad file descriptor
hexdump: fastloadcd: No such file or directory
hexdump: fastloadcd: Bad file descriptor
hexdump: diskload9600: No such file or directory
hexdump: diskload9600: Bad file descriptor
hexdump: diskload9600: No such file or directory
hexdump: diskload9600: Bad file descriptor
hexdump: diskload8000: No such file or directory
hexdump: diskload8000: Bad file descriptor
hexdump: diskload8000: No such file or directory
hexdump: diskload8000: Bad file descriptor
hexdump: diskload2: No such file or directory
hexdump: diskload2: Bad file descriptor
hexdump: diskload2: No such file or directory
hexdump: diskload2: Bad file descriptor
hexdump: diskload3: No such file or directory
hexdump: diskload3: Bad file descriptor
hexdump: diskload3: No such file or directory
hexdump: diskload3: Bad file descriptor
-i used with no filenames on the command line, reading from STDIN.
-i used with no filenames on the command line, reading from STDIN.
-i used with no filenames on the command line, reading from STDIN.
-i used with no filenames on the command line, reading from STDIN.
-i used with no filenames on the command line, reading from STDIN.
-i used with no filenames on the command line, reading from STDIN.
```

Despite this, the build process continues. However, the released version builds only `c2t` (the current master builds both `c2t` and `c2t-96h`.

Using macOS Sierra with command-line tools from the latest Xcode (9.0), and cl65 V2.16 installed with Homebrew (`brew install cc65`).

*Edit:* added cl65 version.

**Comment by datajerk** [comment]
Just updated.  Try again.  I still use 2.13.3, looks like 2.16 may have changed.  I also fixed the perl -i warnings a while ago, please make sure you have the latest source.

**Comment by lifepillar** [comment]
Thanks for the quick fix, the current master builds without errors. Below you find the log: there are several messages, but I guess they are ok.

Would you mind tagging a  point release with these fixes? So I can add a Homebrew formula to [my repo](https://github.com/lifepillar/homebrew-appleii).

```
./makeheader
rm -f autoload diskload2 diskload3 diskload8000 diskload9600 fastload8000 fastload9600 fastloadcd inflate autoload.o diskload2.o diskload3.o diskload8000.o diskload9600.o fastload8000.o fastload9600.o fastloadcd.o inflate.o autoload.lst diskload2.lst diskload3.lst diskload8000.lst diskload9600.lst fastload8000.lst fastload9600.lst fastloadcd.lst inflate.lst autoload.mon diskload2.mon diskload3.mon diskload8000.mon diskload9600.mon fastload8000.mon fastload9600.mon fastloadcd.mon inflate.mon
cl65 -t none --listing  autoload.s 
cl65 -t none --listing  diskload2.s 
cl65: No input files
cl65 -t none --listing  diskload3.s 
cl65: No input files
cl65 -t none --listing  diskload8000.s 
cl65: No input files
cl65 -t none --listing  diskload9600.s 
cl65: No input files
cl65 -t none --listing  fastload8000.s 
cl65: No input files
cl65 -t none --listing  fastload9600.s 
cl65: No input files
cl65 -t none --listing  fastloadcd.s 
cl65: No input files
cl65 -t none --listing  inflate.s 
cl65: No input files
cl65: No input files
hexdump: autoload: No such file or directory
hexdump: autoload: Bad file descriptor
hexdump: autoload: No such file or directory
hexdump: autoload: Bad file descriptor
hexdump: inflate: No such file or directory
hexdump: inflate: Bad file descriptor
hexdump: inflate: No such file or directory
hexdump: inflate: Bad file descriptor
hexdump: fastload9600: No such file or directory
hexdump: fastload9600: Bad file descriptor
hexdump: fastload9600: No such file or directory
hexdump: fastload9600: Bad file descriptor
hexdump: fastload8000: No such file or directory
hexdump: fastload8000: Bad file descriptor
hexdump: fastload8000: No such file or directory
hexdump: fastload8000: Bad file descriptor
hexdump: fastloadcd: No such file or directory
hexdump: fastloadcd: Bad file descriptor
hexdump: fastloadcd: No such file or directory
hexdump: fastloadcd: Bad file descriptor
hexdump: diskload9600: No such file or directory
hexdump: diskload9600: Bad file descriptor
hexdump: diskload9600: No such file or directory
hexdump: diskload9600: Bad file descriptor
hexdump: diskload8000: No such file or directory
hexdump: diskload8000: Bad file descriptor
hexdump: diskload8000: No such file or directory
hexdump: diskload8000: Bad file descriptor
hexdump: diskload2: No such file or directory
hexdump: diskload2: Bad file descriptor
hexdump: diskload2: No such file or directory
hexdump: diskload2: Bad file descriptor
hexdump: diskload3: No such file or directory
hexdump: diskload3: Bad file descriptor
hexdump: diskload3: No such file or directory
hexdump: diskload3: Bad file descriptor
gcc -Wall -Wno-unused-value -Wno-unused-function -I. -O3 -o bin/c2t c2t.c -lm
gcc -Wall -Wno-unused-value -Wno-unused-function -I. -O3 -o bin/c2t-96h c2t-96h.c -lm
```

**Comment by datajerk** [comment]
That output is messed up.  I tested with cl65 2.16 and that is the source of the problems.  The `--listing`|`-l` option now requires a filename parameter.  I fixed the Makefile to address this, however not all the code in this repo will assemble with cl65 2.16.  I do not have a quick fix for this. So at this time cl65 2.13.3 is required.  I updated the `README.md` reflect this.  When I merge c2t and c2t-96h, refactor, and other cleanup I'll try to support the latest cl65 as well.
